js/main.js

```diff
@@ -1,40 +1,45 @@
-// "use strict";
+"use strict";
 
 var pineTree = {};
-
-
-
+var btn = document.getElementById('btn');
+    console.log(btn);
```

Comment:
Before you make a pull request, all these console logs should be deleted. Final projects should only have code and comments where they are necessary to summarize code that needs more explanation.